packages/rancher-sachet/charts/templates/configmap-pre-install.yaml

```diff
@@ -0,0 +1,172 @@
+{{/*This file is applied when the operation is helm install and the target confimap does not exist */}}
+{{- if and (.Release.IsInstall) (not (lookup "v1" "ConfigMap" (include "sachet.namespace" . ) (include "sachet.fullname" .))) }}
```

Comment:
good catch!
My initial thought is to skip this file if it is an upgrade, but it is a `pre-install` hook so it will never be called if it is a `helm upgrade` 

the `.Release.IsInstall` check is removed.